### ISSUE TITLE
Decreased total iron binding capacity

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5950]
**Preferred term label:** Decreased total iron binding capacity

**Synonyms**

**Definition (free text, please give PubMed ID)** A reduction in the total-iron binding capacity, which measures how much serum iron is bound if an excess of radioactive iron is added.

**Parent term (use hpo.jax.org/app)** Abnormality of iron homeostasis HP:0011031

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
